cdhweb/people/wagtail_hooks.py

```diff
@@ -52,11 +53,17 @@ class ProfileAdmin(ThumbnailMixin, ModelAdmin):
     thumb_image_field_name = "image"
 
 
+class LinkTypeAdmin(ModelAdmin):
+    model = RelatedLinkType
+    menu_icon = "link"
+    list_display = ("name", "sort_order")
+
+
 class PeopleGroup(ModelAdminGroup):
     menu_label = "People"
     menu_icon = "group"
     menu_order = 200
-    items = (PersonAdmin, TitleAdmin, ProfileAdmin)
+    items = (PersonAdmin, TitleAdmin, ProfileAdmin, LinkTypeAdmin)
```

Comment:
@haydngreatnews I'm not sure I'm following their request here. Are they just wanting to add new. link types, or are they wanting to change the link's label? What is the effect in the FE?

Reply:
The example given was that they want to be able to add a "LinkedIn" link (but also any other type they fancy)

The `name` is used as the link text, so setting up a `LinkedIn` one creates a link like this
![Screenshot 2024-07-22 at 13-07-04 Akrish Adhikari](https://github.com/user-attachments/assets/6dd47d20-d58a-48b1-9978-4bd1de44b7be)
via this [piece of template logic](https://github.com/springload/cdh-web/blob/2f4b6c21f0add2f9542ca69dfff865c55b10199d/cdhweb/people/templates/people/profile.html#L22)

Reply:
That feels a bit convoluted and like extra steps. I would expect the drop-down to have a change in logic, not a visual change only.

Reply:
Would I be correct in saying that, based on:
> People: When adding a link to our profiles, I thought we were going to be able to name the link whatever we wanted, as to not be limited by a select list. For example, I’d type in “LinkedIn” here instead. 

What you were expecting is for me to change the field on the [RelatedLink](https://github.com/springload/cdh-web/blob/2f4b6c21f0add2f9542ca69dfff865c55b10199d/cdhweb/pages/models.py#L465-L477) model to be a `CharField` instead of this current ForeignKey?
I could do that, but it would introduce a breaking change *and* a data-migration across all the existing link classes. It seems to me that in a place where the bulk of the site has already been through a testing cycle, the simpler change is probably the better one

Reply:
Is it in this context, but without seeing the original request and what the drop-down actually does, it's hard to understand or justify the change :) . Looking at their request, I would've expected to see an additional text field for a link text override. Both a text field and a dropdown make sense in their own context

Could you add some docs around it please?